# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/galan/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- galan-0.2.14/src/event.c.orig	2004-05-13 20:36:28.045674776 +0200
+++ galan-0.2.14/src/event.c	2004-05-13 20:37:54.848478744 +0200
@@ -55,6 +55,7 @@
 		memcpy( dst->d.array.numbers, src->d.array.numbers, src->d.array.len * sizeof(gdouble) );
 		break;
 	    default:
+		;
 	}
     }
 	
@@ -73,6 +74,7 @@
 		    free( e->d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( e );
     }
@@ -91,6 +93,7 @@
 		    free( evq->e.d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( evq );
     }
